CONTRIBUTING.asciidoc

```diff
@@ -526,6 +526,21 @@ generate code and subsequently overwrite part or all of it. Running with all
 will slow Valgrind down noticeably.
 ____
 
+Setting up a Windows Development Environment
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* Install https://www.python.org/ftp/python/3.4.4/python-3.4.4.msi[Python 3.4] (32bit!)
```

Comment:
Huh, I just removed all python slots on my windows machine and installed only python 3.4 and pyqt 5.5 for 64bit and now it works. I'll change the links to the respective download pages that contain both x86 and x86_64 installers as well as source, that's probably better, anyway